taxi/views.py

```diff
@@ -1,9 +1,10 @@
+from django.contrib.auth import get_user
```

Comment:
The `get_user` function is incorrectly imported. It should be `get_user_model` if you intend to get the user model, or you should use `request.user` directly in the view.